.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  release:
+    types: [created]
+
+jobs:
+  # test:
+  #   runs-on: ubuntu-latest
+  #   if: github.event_name != 'release'
+  #   steps:
+  #     - name: Checkout
+  #       uses: actions/checkout@v2
+
+  #     - name: Install Ruby
+  #       uses: ruby/setup-ruby@v1
+  #       with:
+  #         bundler-cache: true
+
+  #     - name: Run tests
+  #       run: |
+  #         bundle exec rspec
```

Comment:
Is the test suite for this gem broken?

Reply:
geminabox-rake won't work with bundler 2, and I didn't want to go down a dependency rabbithole :(